include/elliptics/core.h

```diff
@@ -85,6 +85,12 @@
 
 #define DNET_DEFAULT_CACHE_PAGES_NUMBER 1
 
+#define DNET_DEFAULT_MONITOR_TOP_K 50
```

Comment:
Use more understandable name like `DNET_DEFAULT_MONITIR_TOP_LENGTH` and `DNET_DEFAULT_MONITOR_TOP_MAX_SIZE` and please put comments there.